### PR TITLE
✨ Added Astro Docs for Strapi CMS integration 📚

### PR DESCRIPTION
**Pull request by Elod-T**
🌐 Unlock a world of possibilities for your web development projects by harnessing the power of Astro and the flexibility of Strapi CMS. 🚀

#### What kind of changes does this PR include?

- New or updated content

#### Description

This pull request adds documentation for integrating [Strapi CMS](https://strapi.io/) with Astro. It includes step-by-step instructions and code snippets. The goal is to help developers to leverage Astro and Strapi CMS seamlessly for efficient website development. Let's merge and provide our community with valuable resources for exploring this integration!

### CODE REVIEW

**Review by dreyfus92 — APPROVED** [review]
Awesome job @Elod-T, LGTM! ✅

**Comment by Elod-T** [comment]
> Awesome job @Elod-T, LGTM! ✅

Thank you! 😊

**Comment by sarah11918** [comment]
Hi there @Elod-T! Thanks for giving us a whole 'nother CMS guide! Wow! 🙌 

This looks really comprehensive, and I love the effort to detail here. I think people will really appreciate having something so clear to follow! As you can appreciate, this is a large contribution so it will take a while to go through carefully, test, and edit etc. It will take some time, but I look forward to getting this in the docs!

In the mean time, I would ask you to read our [Writing Guide](https://github.com/withastro/docs/blob/main/WRITING.md) and please follow what it asks for there, especially with respect to voice (e.g. like using "we" and "let's"). It would be helpful if you could make those appropriate edits first before I do more serious editing. 

Secondly, in Astro docs, we typically do not describe how to do the non-Astro parts and stick to only what's needed to connect Astro to other services. It is not practical (and not our job!) for us to describe how do the set up for the external pieces as that is subjet to change. It's an ongoing maintenance burden for us to continue to check to make sure things haven't changed on someone else's end, and really, docs should already exist on their site. You do have some instructions here like setting up a Strapi server that should not be included. It's enough to state what's needed as a prerequisite, and it doesn't add more content on an already long page that people have to get through!

And lastly, these CMS guide recipes would ideally be written using only official packages and tools from the Astro and the external party itself (ie Strapi). Again, from a maintenance standpoint, the more external dependencies included, especially from entirely other sources, the more work it is to ensure that all instructions are accurate and up to date. If someone stops maintaining a project, or makes a breaking change, then we typically wouldn't know to update (as opposed to our own stuff), and our guide suffers. So, if you can write a guide that doesn't use e.g. the external Markdown package to render the content, then it's more future-proof.

Side note: I'm not sure whether you're aware, but there is some pre-existing Strapi + Astro content out on the web that is not at all accurate or easy to follow, and there are several complaints about it not working and a very angry community response.  😅 Therefore, we will have to be particularly careful here as I suspect this will be a resource people will find after being frustrated with other sources. We have to assume that our readers might not be in the mood to give us the benefit of the doubt, so we have to have to get this one right!

We're going to have to have a particularly high standard with any Strapi content officially in our docs, so buckle up, and enjoy the ride!  :rocket:

**Comment by Elod-T** [comment]
Hello @sarah11918 and thank you for your response! I'll make the necessary edits to adhere to the writing guide, adjust the voice accordingly, and remove any unrelated instructions.

**Review by ElianCodes — APPROVED** [review]
Awesome! Thank you!

**Review by sarah11918 — COMMENTED** [review]
Hey @Elod-T! Whew!  :sweat_smile: 

I just pushed a commit with a bunch of text edits to tighten/clean some of the writing up. Of course, I was mostly looking at this as an editor, and not as someone who knows Strapi! So, now it's your turn to review this and make sure that it still is accurate and makes sense.

When I edit, I try to make wording direct, clear and concise. That often means chopping, and sometimes a key point or nuance can get lost. PLEASE DO review this carefully to make sure the content is still appropriate! Put suggestions where anything needs to change, and we can negotiate the balance between writing and content.  :raised_hands:

**Comment by Elod-T** [comment]
Hi @sarah11918, awesome work there!
The guide looks clean and easy to follow. The only thing I could find is that the import paths were incorrect in a file.

File: src/content/docs/en/guides/cms/strapi.mdx [hunk 1]
```diff
@@ -185,10 +185,10 @@ You can modify this interface, or create multiple interfaces, to correspond to y
 
 3. Using data from the `articles` array returned by the API, display your Strapi blog posts in a list. These posts will link to their own individual pages, which you will create in the next step.
 
-  ```astro title="src/pages/blog/index.astro"
+  ```astro title="src/pages/index.astro"
```
Comment (sarah11918) [inline src/content/docs/en/guides/cms/strapi.mdx:188]: Perfect, thanks for catching this!

**Comment by sarah11918** [comment]
Thanks @Elod-T ... let's do this thing! Thanks for being a great partner to work with, and welcome to Team Docs!  🥳

**Comment by Elod-T** [comment]
Thank you for your help and kindness @sarah11918!
It's been great working with you!

**Pull request merged**